search-app/src/cmr/search/services/generic_association_service.clj

```diff
@@ -25,8 +25,8 @@
 (defn- context->user-id
   "Returns user id of the token in the context. Throws an error if no token is provided"
   [context]
-  (if-let [token (:token context)]
-    (util/lazy-get context :user-id)
```

Comment:
Put this space back in, both branches of if should be at same indentation

Reply:
fixed